README.md

```diff
@@ -327,7 +327,7 @@ natural algebraic approach.
 using JuMP, Gurobi
 
 # pass params as keyword arguments to GurobiSolver
-m = Model(solver=GurobiSolver(Presolve=0))
+m = Model(with_optimizer(Gurobi.Optimizer, Presolve=0))
```

Comment:
`m` -> `model` according to JuMP style guide

Reply:
Done.